.gitignore

```diff
@@ -1 +1,2 @@
 src/
+tags
```

Comment:
This is not needed for this repo, right?
Since there are no binaries, I would suggest removing the `.gitignore` file instead.